scripts/compile-sass.js

```diff
@@ -1,91 +1,39 @@
 /* eslint-disable no-console */
 
 import { join, resolve } from 'node:path';
-import { pathToFileURL } from 'node:url';
 
 import autoprefixer from 'autoprefixer';
 import chalk from 'chalk';
 import fs from 'fs-extra';
 import postcss from 'postcss';
-import { compile } from 'sass';
+import { initAsyncCompiler, NodePackageImporter } from 'sass-embedded';
 
 const __dirname = import.meta.dirname;
 const baseDir = resolve(__dirname, '..');
 const inDir = join(baseDir, 'src', 'scss');
 const outDir = join(baseDir, '_built', 'css');
 const destCssDir = join(baseDir, 'assets', 'css');
 
-let pnp;
+const compiler = await initAsyncCompiler();
```

Comment:
@jamesnw Average build times were just under 1s with `sass` using either a compiler or top-level methods. After switching to `sass-embedded`, times dropped to ~650ms using top-level methods, then down to ~450ms using `initCompiler`, and now down to ~300ms using `initAsyncCompiler`.